language-antlr-utils/src/main/java/de/jplag/antlr/AbstractAntlrParserAdapter.java

```diff
@@ -25,40 +25,55 @@
  * @param <T> The type of the antlr parser
  */
 public abstract class AbstractAntlrParserAdapter<T extends Parser> extends AbstractParser {
+
+    private final boolean extractsSemantics;
+
+    /**
+     * New instance
+     * @param extractsSemantics If true, the listener will extract semantics along with every token
+     */
+    protected AbstractAntlrParserAdapter(boolean extractsSemantics) {
+        super();
```

Comment:
Default constructor call can be omitted.

Reply:
This is style :D .. I personally would always keep it. But this should be consistent in JPlag :)